app/src/main/java/com/example/trackflix/fragments/update/UpdateFragment.kt

```diff
@@ -84,8 +82,18 @@ class UpdateFragment : Fragment() {
             updateTrackable()
         }
 
-        //addDeleteOptionsMenu
-        setHasOptionsMenu(true)
```

Comment:
Warum kommt die delete Option hier weg?

Reply:
Das hab ich in das UpdateFragmentPager Fragment gschoben weil der Viewpager sozusagen die Fragmente beinhaltet und da beim ausführen mit dem Menu im Update Fragment der delete button komisch kopiert worden ist und war ziemlich buggy. Deswegen der delete Button in der Bar über dem Viewpager. Dann is der nur einmal voehanden und wenn gelöscht wird holt er sich des aktuelle Element aus der Liste und löscht das